components/engine_test/src/lib.rs

```diff
@@ -129,13 +130,15 @@ pub mod kv {
             db_opt: Option<DBOptions>,
             cfs: &[&str],
             opts: Option<Vec<CFOptions>>,
+            multi_rocksdb: bool,
```

Comment:
Why the change? If it's single rocksdb, then it should use `KvEngineFactory`.

Reply:
KvEngineFactory uses RocksEngine whereas TestTabletFactory uses KvTestEngine.

Reply:
Then you can just implement `TestTabletFactory` and `TestTabletFactoryV2` just like `KvEngineFactory` and `KvEngineFactoryV2`.

Reply:
For the consistency with real implementation? Okay, I will do it later.